CreeDictionary/search_quality/sample.csv

```diff
@@ -0,0 +1,64 @@
+English,Nêhiyawêwin 1,Nêhiyawêwin 2,Nêhiyawêwin 3,,,Poor result,Notes
+star,acâhkos,atâhk,,,,,
+see,wâpamêw,wâpiw,wâpahtam,,,kwêyask,"Ignore ""\[see"""
+eat,mîcisow,mîciw,mowêw,,,,
+fire,iskotêw,,,,,,
+happy birthday,miyo-tipiska,,,,,,
+thank you,ay-hay,hay-hay,,,,,
+dog,atim,,,,,,very irregular inflections
+horse,misatim,,,,,,
+'horse',misatim,,,,,,
+"""horse""",misatim,,,,,,
+beaverberry,amiskomin,amiskominak,,,,,
+beaverberries,amiskomin,amiskominak,,,,,
+beaver-berry,amiskomin,amiskominak,,,,,
+beaver-berries,amiskomin,amiskominak,,,,,
+beaver berry,amiskomin,amiskominak,,,,,
+beaver berries,amiskomin,amiskominak,,,,,
+berries,mînisa,mînis,,,,,
+berry,mînis,mînisa,,,,,
+berri,mînis,mînisa,,,,,
+stove pipe,okohtaskwahikan,,,,,,
+stove-pipe,okohtaskwahikan,,,,,,
+stovepipe,okohtaskwahikan,,,,,,
+in the area,cîkâhtaw,,,,,,
+bread,pahkwêsikan,,,,,,
+yellow hat,osâwastotin,,,,,,
+sleep,nipâw,,,,,,
+I love you,kisâkihitin,,,,,,
+this,ôma,awa,ôhi,,,,
+that,anima,,,,,,
+good morning,miyo-kîkisêpâyâw,,,,,,
+me,niya,,,,,,
+good bye,ekosi mâka ,,,,,,
+goodbye,ekosi mâka ,,,,,,
+work together,mâmawi-isîhcikêwin,,,,,,
+working together,mâmawi-isîhcikêwin,,,,,,
+work togeth,mâmawi-isîhcikêwin,,,,,,
+Cree,nêhiyaw,,,,,,
+Edmonton,amiskwaciy-wâskahikan,,,,,,
+edmonton,amiskwaciy-wâskahikan,,,,,,
+beaver,amisk,,,,,,
+hill,waciy,,,,,,
+house,wâskahikan,,,,,,
+beaver hill,natimîwiyiniwak,amiskwacîwiyiniwak,,,,,
+beaver hills,natimîwiyiniwak,amiskwacîwiyiniwak,,,,,
+Calgary,otôskwanihk,,,,,,
+calgary,otôskwanihk,,,,,,
+cat,minos,,,,,,
+car,pôsîw,pôsiy,,,,,
+rabbit,wâpos,,,,,,
+water,nipiy,,,,,,
+telling a story,âcimêw,,,,,,
+on water,nipîhk,,,,,,
+hello,tânisi,,,,,,
+difficult thing,âyiman,,,,,,
+many snakes,kinêpikoskâw,,,,,,
+dances,nîmiw,,,,,,
+i see you,kinêpikoskâw,,,,,,
+he sees him,wâpamêw,,,,,,
+she sees him,wâpamêw,,,,,,
+starblanket,atâhkakohp,acâhkosa kâ-otakohpit,,,,,
+star blanket,atâhkakohp,acâhkosa kâ-otakohpit,,,,,
+being taught,kiskinwahamâkosiw,,,,,,
+they see us,niwâpamikonânak,kiwâpamikonawak,,,,,
```

Comment:
@aarppe @katieschmirler @atticusha Please review this list and make suggestions!

@andrewdotn: is there a documented format for this file? Also, do you know how to get GitHub to render a pretty-printed version of this. I know it's capable!

Reply:
Click the `CreeDictionary/search_quality/sample.csv` label in the conversation view to go to https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/pull/635/files/c35622e5cdae3b6ca0a17ccf99dd1257589ee7fd#diff-1048d6e8db6891c0b02807610cba69b649b00ecd35517d5b8629211c8570fd1a

Then click the “…” menu → View file to go to https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/blob/c35622e5cdae3b6ca0a17ccf99dd1257589ee7fd/CreeDictionary/search_quality/sample.csv

Reply:
When *everyone* asked what the column headings meant, I took that as a signal that they were poorly named. I have renamed the first column to ‘Query’, and have added a `README.md` file that explains them in more detail.